docs/Data_Portal/Users_Guide/proteinpaint_lollipop_quick_start

```diff
@@ -0,0 +1,148 @@
+
+## ProteinPaint Features
+
+When launched, ProteinPaint will display a search box where users can enter a gene symbol, alias, or GENCODE accession. Once a gene is entered, a lollipop frame is displayed with the name of the chart in the header.
+
+[![Lollipop Frame](images/lollipop2.png)](images/lollipop2.png "Click to see the full image.")
+
+There are three main panels in the ProteinPaint tool: [search box](#Search-Box), [Lollipop chart](#Lollipop-Chart-Panel), and [legend](#Legend-panel).
+
+[![3 Main Panels](images/lollipop3.png)](images/lollipop3.png "Click to see the full image.")
+
+### Search Box
+
+The search box allows users to enter a gene symbol, GENCODE accession, or RefSeq accession. All gene symbols are based on the [HGNC](https://www.genenames.org/about/guidelines/) guidelines. 
+
+[![Search Box](images/lollipop4.png)](images/lollipop4.png "Click to see the full image.")
+
+When searching for a gene, the text box will auto-complete with possible matches. If a gene is incorrectly entered, the search box will display an error. 
+
+[![Search Box Example](images/lollipop5.png)](images/lollipop5.png "Click to see the full image.")
+
+### Lollipop Chart Panel
+
+After entering a gene, the tool will display a Lollipop chart for the GDC variants as well as a Protein View for the default isoform. 
+
+#### Lollipop Chart
+
+In the Lollipop chart, the circular discs for each variant are color coded per GDC mutation classes and are proportional to the number of occurrences. Variants in the same position are arranged in descending order of magnitude.
+
+[![Lollipop Chart](images/lollipop10.png)](images/lollipop10.png "Click to see the full image.")
+
+Exon variants report the amino acid change at the referenced codon. For example, G12D is a G > D substitution at the 12th codon of the protein.
+
+The default isoform will appear directly to the right of the gene name. Clicking on the isoform number will open a display to view all other isoforms. From this window, users have the options to switch the display [track](#protein-view) and isoform.
+
+[![Display All Isoforms](images/lollipop8.png)](images/lollipop8.png "Click to see the full image.")
+
+To the left of the Lollipop, users can click on the number of variants and number of samples links to reveal detailed annotations.
+
+[![Sample and Variant Annotations](images/lollipop11.png)](images/lollipop11.png "Click to see the full image.")
+
+Clicking on the number of variants link opens a menu where users can view annotations and manipulate the Lollipop:
+
+* __List:__ Display all variants, each of which can be selected to launch the annotation table which displays consequence, mutation, sample submitter_id, and other data related to the sample
+    * __Mutation:__ Clicking on the mutation hyperlink will launch the [GDC Mutation Summary Page](mutation_frequency.md/#gene-and-mutation-summary-pages)
```

Comment:
the various links to other pages aren't working for me. I think it's because of the `/` after `md` -- makes it seem like, e.g. `mutation_frequency.md` is the name of a folder

Reply:
Fixed!